src/compile/unit.ts

```diff
@@ -320,51 +315,43 @@ export class UnitModel extends Model {
     return UNIT_CHANNELS;
   }
 
-  protected mapping() {
-    return this.encoding();
-  }
-
-  public stack(): StackProperties {
-    return this._stack;
+  protected getMapping() {
```

Comment:
I don't like unnecessary `get*` prefix for these function.  It makes code longer everywhere.